packages/boot/test/bootstrapTests/lca.test.ts

```diff
@@ -59,8 +62,8 @@ test.serial('stakeBld', async t => {
     },
   });
 
-  const current = await wd.getCurrentWalletRecord();
-  const latest = await wd.getLatestUpdateRecord();
+  const current = wd.getCurrentWalletRecord();
+  const latest = wd.getLatestUpdateRecord();
```

Comment:
synchronous IO feels funny. But I guess the wallet driver API is that it doesn't do IO here?

Reply:
right, what it's reading is local